CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Origin (DCO). This document was created by the Linux Kernel community and is a
 simple statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
 
-Any copyright notices in this repo should specify the authors as "the Loodse XXX project contributors".
+Any copyright notices in this repo should specify the authors as "the Loodse Machine Controller project contributors".
```

Comment:
Either this should be changed to `the Machine Controller Authors`, or the copyright for all files will need to be updated :grimacing:

Reply:
Hmm, KubeOne uses the term "project contributors". IMHO the renaming of all headers should be an extra issue.

Reply:
Cool, I can follow-up on that.